### PR TITLE
Load current run in muon analysis

### PR DESCRIPTION
**Pull request by AnthonyLim23**
**Description of work.**
The load current run button in (new) muon analysis should work. 

**To test:**
Open muon analysis 2
Click load current run
Should load some data (for ISIS instruments)

No associated issue 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: scripts/test/Muon/utilities/load_utils_test.py [hunk 1]
```diff
@@ -75,6 +76,15 @@ def test_load_workspace_from_filename_for_existing_file(self):
         self.assertAlmostEqual(load_result['TimeZero'], 0.55000, 5)
         self.assertEqual(run, 22725)
 
+    def test_load_workspace_from_filename_for_file_path(self):
+        mock_alg = mock.MagicMock()
+        filename = 'PSI'+os.sep + 'run_1529_templs0.mon'
```
Comment (giovannidisiena) [inline scripts/test/Muon/utilities/load_utils_test.py]:
```suggestion
        filename = 'PSI'+ os.sep + 'run_1529_templs0.mon'
```

**Review by giovannidisiena — CHANGES_REQUESTED** [review]
Functional testing passed, minor nitpick then ready to be approved

**Comment by AnthonyLim23** [comment]
@giovannidisiena I think that I have fixed your comment.

**Review by giovannidisiena — APPROVED** [review]
:shipit:

File: scripts/test/Muon/utilities/load_utils_test.py [hunk 2]
```diff
@@ -75,6 +76,15 @@ def test_load_workspace_from_filename_for_existing_file(self):
         self.assertAlmostEqual(load_result['TimeZero'], 0.55000, 5)
         self.assertEqual(run, 22725)
 
+    def test_load_workspace_from_filename_for_file_path(self):
+        mock_alg = mock.MagicMock()
+        filename = 'PSI'+ os.sep + 'run_1529_templs0.mon'
```
Comment (DanNixon) [inline scripts/test/Muon/utilities/load_utils_test.py]: A cleaner alternative here would have been `os.path.join`.

**Pull request merged**